components/selectors/parser.rs

```diff
@@ -1201,6 +1201,8 @@ pub mod tests {
         type PseudoElement = PseudoElement;
     }
 
+    type ShutUpTidy = String;
```

Comment:
(Tidy complains when it sees `&String` as a function argument, even if in this case is unavoidable because it's generic code).